docs/dapps/nfts/intro-to-erc721s.md

```diff
@@ -0,0 +1,140 @@
+
+The Base URI field listed here is the URL of the metadata folder uploaded to Pinata(for example, ours is `https://gateway.pinata.cloud/ipfs/QmYdWxbiwsfsYcW1CYQPgYujAc9FMLPG3fgFcxFskbSsFa`). Paste that into the Base URI field. After the Wizard adds our variables to the template, our contract should look like this:
+
+![Contract Wizard Populated](intro-to-erc721s/3-wizard-populated.png)
+
+Next, we will want to check the `Mintable` and `Auto Increment Ids` boxes. This will populate a mint function into our template that would handle the incrementation of token Ids on mint if we had more than one NFT in our collection. We still want it to auto-assign our 1 NFT, so we will check it.
+
+This automatically checks the `Ownable` button, which gives the safeMint function the `onlyOwner` modifier. This modifier indicates that only the owner of the smart contract will be able to successfully call the function.
+
+:::info
+**Note:** This modifier should be removed when creating a smart contract for a public mint. Otherwise, users would not be able to successfully mint the NFTs when calling the safeMint function. This tutorial only handles the owner’s wallet address, so it is being left in.
+:::
+
+Now, our contract is a little more populated:
+
+![Contract Wizard SafeMint](intro-to-erc721s/4-wizard-safemint.png)
+
+For this simple example, we will not add any additional functionality to the `safeMint` function. Currently, it mints one NFT to the address specified in the function call. There is no cost to mint the NFT other than the gas fee for the transaction itself. 
+
+This `safeMint` function currently doubles as an airdrop function because the address the NFT is minted to does not need to be the function owner. This functionality becomes very useful when NFT collection owners want to give away NFTs for free outside of the normal minting window.
+
+At this point, our smart contract is ready. At the top, you can click `Open in Remix` to get ready to deploy your smart contract.
+
+![Contract Wizard Open Remix](intro-to-erc721s/5-open-remix.png)
+
+## Deploying the Smart Contract with Remix
+
+[Remix IDE](https://remix.ethereum.org/) is a solidity compiler that allows you to edit, compile, and deploy your smart contract. This will prevent you from needing to download any other coding environments at this stage.
+
+Once you've imported your contract, the first thing you need to do is compile it. Hit the `Compile` button on the left-hand side. You could also use the keyboard shortcut `Ctrl / Command + S`.
+
+![Remix Compile](intro-to-erc721s/6-remix-compile.png)
+
+Once completed, you will get a green checkmark on the far left tab and will see options to Publish on Ipfs or Swarm. Those are not important to our tutorial. Next, you will click on the bottom tab on the left-hand side to move to the deployment page.
```

Comment:
Consider using contractions to make the text read more casually.

Ex: "...you'll get a green checkmark..." instead of "...you will get a green checkmark..." and "you'll click on the bottom tab" instead of "you will click on the bottom tab"